src/shared/logger.ts

```diff
@@ -2,21 +2,18 @@ import chalk from "chalk";
 
 export const error = (err: Error | string, code = 0) => {
   if (process.env.NODE_ENV === "test") return;
-  if (err instanceof Error) {
-    console.error(err);
-  } else {
-    console.error(chalk`{red.bold ERROR:} ${err}`);
-  }
+  const text = err instanceof Error ? err : chalk.red.bold(`ERROR: ${err}`);
+  console.error(text);
   console.log(
     "If you think this is a bug, you can report it: https://github.com/benawad/destiny/issues"
   );
-
   process.exit(code);
 };
 
 export const info = (msg: string) => {
   if (process.env.NODE_ENV === "test") return;
-  console.info(chalk`{green.bold INFO:} ${msg}`);
+  const text = chalk.green.bold(`INFO: ${msg}`);
```

Comment:
Is this wanted that it's logging the full message in green and not only the prefix `INFO:` (same for the `warn`)?
I think this is a bit too much...

![](https://i.gyazo.com/970e1efcc2c93083ca52a478be8846cb.png)

Reply:
I see what you mean. It's definitely too much, I'll change it now.